docs/Configure the sample site.md

```diff
@@ -3,27 +3,29 @@
 
 Follow these steps to configure the sample site before running it.
 
-1. You must have already have an active Dynamics 365 Fraud Protection account. If you do not, please stop and contact Dynamics 365 Fraud Protection or your system integration partner.
+1. You must already have an active Dynamics 365 Fraud Protection account. If you do not, please stop and contact Dynamics 365 Fraud Protection or your system integration partner.
 1. If you haven't already, clone this repository locally.
 1. In your local repository, open [appsettings.json](../src/Web/appsettings.json).
 1. Go to the [Dynamics 365 Fraud Protection integration portal](https://dfp.microsoft-int.com).
+   1. If your Azure global administrator has not already visited the portal, please ask them to do so first. They will need to agree to the terms of use in order to setup the integration environment.
 1. Sign in and you will be on the dashboard.
+1. Gather the following pieces of information from the dashboard:
+   1. Copy the "Instance ID" GUID and set the "DeviceFingerprintingCustomerId" setting in your appsettings.json file to it.
+   1. Copy the "Directory ID" GUID and set the "Authority" setting in your appsettings.json file to "https://login.microsoftonline.com/[Directory ID]".
```

Comment:
Appears weird on read mode, wither we should remove it being hyperlink or have complete url as hyperlink.

Reply:
Good point.  I've added an '_' (underscore) in between so it all links together.